test/end-to-end/employees/registration.page.js

```diff
@@ -0,0 +1,170 @@
+        FU.uiSelect('EmployeeCtrl.employee.service_id', service);
+    }
+
+    // set grade
+    function setGrade (grade){
+        FU.uiSelect('EmployeeCtrl.employee.grade_id', grade);
+    }
+
+    // set fonction
+    function setFonction (fonction){
+        FU.uiSelect('EmployeeCtrl.employee.fonction_id', fonction);
+    }
+
+    // set email
+    function setEmail (email){
+        FU.input('EmployeeCtrl.employee.email', email);
+    }
+
+    // set address
+    function setAddress (address){
+        FU.input('EmployeeCtrl.employee.adresse', address);
+    }
+
+    // set hospital Number
+    function setHospitalNumber(hn){
+        FU.input('EmployeeCtrl.employee.hospital_no', hn);
+    }
+
+    // set debtor group
+    function setDebtorGroup (dg){
+        FU.uiSelect('EmployeeCtrl.employee.debtor_group_uuid', dg);
+    }
+
+    // set creditor group
+    function setCreditorGroup (cg){
+        FU.uiSelect('EmployeeCtrl.employee.creditor_group_uuid', cg);        
+    }
+
+    // set bank
+    function setBank (bank){
+        FU.input('EmployeeCtrl.employee.bank', bank);
+    }
+
+    // set bank account
+    function setBankAccount (bankAccount){
+        FU.input('EmployeeCtrl.employee.bank_account', bankAccount);
+    }
+
+    // set origin location
+    function setOriginLocation(locations){
+        components.locationSelect.set(locations, 'origin-location-id');    
+    }
+
+    // set current location
+    function setCurrentLocation(locations){
+        components.locationSelect.set(locations, 'current-location-id');    
+    }
+    
+    function isEmpoyeeCreated (resp){
+        return FU.exists(by.id('receipt-confirm-created'), resp);
+    }
+
+    function requiredFIeldErrored (){
+        FU.validation.error('EmployeeCtrl.employee.display_name');
+        FU.validation.error('EmployeeCtrl.employee.sexe');
+        FU.validation.error('EmployeeCtrl.employee.code');
+        FU.validation.error('EmployeeCtrl.employee.grade_id');
+        FU.validation.error('EmployeeCtrl.employee.creditor_group_uuid');
+        FU.validation.error('EmployeeCtrl.employee.debtor_group_uuid');
+        FU.validation.error('EmployeeCtrl.employee.dob');
+        FU.validation.error('EmployeeCtrl.employee.hospital_no');
+    }
+
+    function noRequiredFieldOk(){
+        FU.validation.ok('EmployeeCtrl.employee.locked');
+        FU.validation.ok('EmployeeCtrl.employee.nb_spouse');
+        FU.validation.ok('EmployeeCtrl.employee.nb_enfant');
+        FU.validation.ok('EmployeeCtrl.employee.phone');
+        FU.validation.ok('EmployeeCtrl.employee.email');
+        FU.validation.ok('EmployeeCtrl.employee.bank');
+        FU.validation.ok('EmployeeCtrl.employee.bank_account');
+        FU.validation.ok('EmployeeCtrl.employee.service_id');
+        FU.validation.ok('EmployeeCtrl.employee.fonction_id');
+        FU.validation.ok('EmployeeCtrl.employee.adresse');
+    }
+
+    page.createEmployee = createEmployee;
```

Comment:
This file has 22 lines of `page.X = X`.  In ES6, you can use a [class](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes) to make the code read better.  Instead of this:
```js
function SomePage() {
  const page = this;

  setSomeProperty(value) {
    FU.input('$ctrl.key', value);
  }

  page.setSomeProperty = setSomeProperty;
}
```
... your code can be written as:
```js
class SomePage {
  setSomeProperty(value) {
    FU.input('$ctrl.key', value);
  }
}
```

In this case, the end behavior will be the same.